src/cmd/run.js

```diff
@@ -168,7 +242,12 @@ export default function run(
         'of Firefox (you need Firefox 49 or higher). For older Firefox ' +
         'versions, use --pre-install');
     }))
-    .then(({firefox, profile, client, addonId}) => {
+    .then((config) => {
+      const firefox: FirefoxProcess = config.firefox;
+      const profile: FirefoxProfile = config.profile;
+      const client: RemoteFirefox = config.firefox;
+      const addonId: string = config.addonId;
```

Comment:
all these type hints makes the code highly verbose. If they aren't necessary, I'd like to avoid them. If they are necessary, maybe we need to refactor this code to pass around a class instance instead (in a later patch).